### DISCUSSION TITLE
职场上领导们的套路 | /posts/taolu/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#610] [Announcements]
https://yuanfan.rbind.io/posts/taolu/

**Comment by botsp** [comment]
领导：你知道为什么我们部门就给你涨吗？

C：是因为我的薪资好多年没涨过了吗？

领导：这倒不是，好多人工作好多年薪水都没动过，涨薪水可不是轮流来的。

C：那是因为去年我的年终考核得了卓越（前5%）吗？

领导：呃……也不是。

C：那是为撒呢？

>>>为啥感觉C是在故意呛领导呢，哈哈

**Reply by earfanfan** [reply]
天啦噜，快给我讲讲是怎么以为 C 是故意呛领导的？哈哈哈，这个 C 就是我，我写出来之前从来没有想到过领导的脑回路还可以是这样的。

**Comment by botsp** [comment]
哎呀，先恭喜你涨薪了~

先说自己工资好多年没涨>>>暗示领导你咋这么不给力呢
然后讲讲自己去年的卓越考核>>>好多人没涨不代表我不能涨啊

然后来一句杀人诛心--"那是为撒呢？"--反正我就该涨了，跟你是没啥关系.

领导眼巴巴等你表扬呢，哈哈哈哈。(瞎想啊，可能更多还要看氛围还有你们的相处模式了，

A/C/F不会都是你吧，，，emmmm

**Reply by earfanfan** [reply]
哈哈，谢谢你。A 不是我，是坐我对面的同事。其实本来是按顺序写了 A/B/C/D/E/F 的故事，写完感觉其中一部分否定和打压太直接，没有撒可总结的套路，于是删掉了。

嗯，长知识了。我一向知道咱们是高语境文化，但自己是个中低手，本来写出来的时候回忆是非常不美好滴，但你这样一解读，我忽然感觉自己好像还挺厉害的呢，哈哈哈，原来我也气到领导了，估计往后几天想起来都能乐呵好一会。以前都是为“中译中”而感到烦恼，长这么大头一次感受到了“中译中”的乐趣。

**Reply by botsp** [reply]
俺也是个中低手，只是脑子中毒太久、做阅读理解做习惯了。

想想真羡慕啊，既涨了薪又顺带反手一刀挥向老板 :sob:

**Reply by earfanfan** [reply]
我上学的时候做阅读理解总结中心思想撒的就很菜，去年过年翻到堂弟的九年级寒假作业，那上面的阅读理解题现在看了依然觉得脑壳疼。至于日常生活中，我一直觉得应该有很多阅读理解题都没正确理解，啊，应该等我到了四十岁的时候自然就会量变形成质变了吧。

话说其实涨薪水是去年的事了，哈哈，想到无意中刀了领导，我的嘴角咧上去就没下来过。

**Reply by botsp** [reply]
双份快乐~

**Reply by earfanfan** [reply]
@botsp 

哈哈，文中的好友姚说她看完评论区也觉得有点快乐，哈哈哈，我又想起来还是觉得快乐，这下是持续的多份快乐了。

**Comment by hongtaoh** [comment]
我一直在做学生。说实话现在让我回国工作我会很害怕，觉得无法适应高强度的工作环境。

**Reply by earfanfan** [reply]
嗯，要是让我去面对一些未知的事情也是会害怕的，不过国内的工作环境并不全都是高强度的，也有正常不用加班的。博客中我堂妹面对的那种环境在我看来就是极其不正常的。
